src/components/auction/OrderbookContainer/index.tsx

```diff
@@ -0,0 +1,166 @@
+      props.margin ? (typeof props.margin !== 'string' ? props.margin[1] : props.margin) : '0'};
+  }
+`
+
+const SectionTitle = styled(PageTitle)`
+  margin-bottom: 0;
+  margin-top: 0;
+`
+
+const StyledButtonSelect = styled(ButtonSelect)`
+  height: 19px;
+  font-size: 13px;
+  border-color: ${({ theme }) => theme.textField.color};
+  padding: 2px 6px;
+  line-height: 13px;
+  cursor: pointer;
+  svg {
+    width: 8px;
+    margin-left: 7px;
+  }
+`
+
+const StyledDropdown = styled(Dropdown)`
+  margin-left: 16px;
+  @media (min-width: 768px) {
+    margin-left: 0;
+    margin-right: 16px;
+  }
+  &.isOpen {
+    button {
+      background-color: ${({ theme }) => theme.textField.color};
+      border-color: ${({ theme }) => theme.textField.color};
+      color: ${({ theme }) => theme.textField.backgroundColor};
+      .fill {
+        fill: ${({ theme }) => theme.textField.backgroundColor};
+      }
+    }
+  }
+  .dropdownItems {
+    min-width: 82px;
+  }
+`
+
+const StyledDropdownItem = styled(DropdownItem)`
+  padding: 5px 13px;
+  font-size: 13px;
+  font-weight: normal;
+  line-height: 1.08;
+  justify-content: space-between;
+`
+
+const StyledCheckbox = styled(Checkbox)`
+  border-radius: 50%;
+  background-color: ${({ theme }) => theme.textField.color};
+  &:before {
+    border-radius: 50%;
+  }
+`
+
+export const OrderBookContainer = (props) => {
+  const { auctionIdentifier, auctionState, derivedAuctionInfo } = props
+
+  const [isChartVisible, setChartVisibility] = useState(true)
+  const dropdownMockData = ['0.001', '0.01', '0.1', '1', '10', '50', '100']
+  return (
+    <>
+      <Wrap alignItems={['flex-start', 'center']} flexDir={['column', 'row']} margin={'0 0 16px 0'}>
+        <SectionTitle as="h2">
+          {(derivedAuctionInfo.auctionState === AuctionState.ORDER_PLACING ||
+            derivedAuctionInfo.auctionState === AuctionState.ORDER_PLACING_AND_CANCELING) &&
+            'Orderbook'}
+        </SectionTitle>
+        <Wrap flexDir={['row-reverse', 'row']} margin={['20px 0', '0']}>
+          {!isChartVisible && (
+            <StyledDropdown
+              dropdownButtonContent={<StyledButtonSelect content={'Granularity'} />}
+              dropdownPosition={DropdownPosition.right}
+              items={dropdownMockData.map((item, index) => (
+                <StyledDropdownItem key={index}>
+                  {item}
+                  <StyledCheckbox />
+                </StyledDropdownItem>
+              ))}
+            />
+          )}
+          <ButtonToggle
+            activate={isChartVisible}
+            left={{ icon: <ChartIcon />, label: 'Chart' }}
+            onClick={() => setChartVisibility(!isChartVisible)}
+            right={{ icon: <TableIcon />, label: 'List' }}
+          />
+        </Wrap>
+      </Wrap>
+      {auctionState === AuctionState.PRICE_SUBMISSION && (
+        <AuctionPending>Auction closed. Pending on-chain price-calculation.</AuctionPending>
+      )}
```

Comment:
This information used to be on the "claimer side" and not on the" orderbookcontainer side". Do you think it makes more sense on the orderbook side?

Reply:
done